cli/smartcontract/smart_contract.go

```diff
@@ -46,6 +47,10 @@ var (
 		Name:  "wallet, w",
 		Usage: "wallet to use to get the key for transaction signing",
 	}
+	walletConfigFlag = cli.StringFlag{
```

Comment:
...then we can reuse the flag from the `cli/wallet` package...

Reply:
They have different usage texts, and this one is invocation-related. They also have different styles of usage text (capitalization). And we don't have any other exported flags, so it doesn't look goot to me. Maybe we can create a separate helper package and move `walletConfigFlag` and `walletFlag` there?